examples/generative/fashionmnist_autoencoder.exs

```diff
@@ -31,9 +38,9 @@ defmodule FashionMNIST do
   defp transform_images({bin, type, shape}) do
     bin
     |> Nx.from_binary(type)
-    |> Nx.reshape({elem(shape, 0), 1, 28, 28})
-    |> Nx.divide(255.0)
-    |> Nx.to_batched(32)
+    |> Nx.reshape({elem(shape, 0), @image_channels, @image_side_pixels, @image_side_pixels})
```

Comment:
```suggestion
    |> Nx.reshape({elem(shape, 0), 1, 28, 28}, names: [:batch, :channels, :width, :height])
```

not an _actual_ suggestion/review, just so you know of this other way of imposing semantics to shapes